pkg/loghttp/push/push.go

```diff
@@ -188,6 +189,12 @@ func ParseRequest(logger log.Logger, userID string, maxRecvMsgSize int, r *http.
 	}
 	linesReceivedStats.Inc(totalNumLines)
 
+	forwardedHeader := r.Header.Get("X-Forwarded-For")
+	agentIP := forwardedHeader
+	if strings.Contains(forwardedHeader, ",") {
+		agentIP = strings.Split(forwardedHeader, ",")[0]
+	}
```

Comment:
I think this could be simplified to:

```suggestion
	agentIP = strings.Split(forwardedHeader, ",")[0]
```
from the split docs `If s does not contain sep and sep is not empty, Split returns a slice of length 1 whose only element is s. `
If there is no comma it will just use the value, if there is we'll return the first

Reply:
Sure, I tested that and it definitely works, so we can use it.  In general, I get nervous with array indexing, but since the docs confirm, I'm good :+1: